include/swift/AST/Expr.h

```diff
@@ -4666,16 +4664,19 @@ class UnresolvedPatternExpr : public Expr {
 class EditorPlaceholderExpr : public Expr {
   Identifier Placeholder;
   SourceLoc Loc;
+  SourceLoc TrailingAngleBracketLoc;
```

Comment:
Here is an example of something that could be ripped out into a separate patch. This could be sliced off.

Reply:
Yes, agreed! Our comments crossed in the ether.